draft-ietf-quic-transport.md

```diff
@@ -727,16 +732,16 @@ Connection ID.
 
 ### Tokens
 
-If the client has a token received in a NEW_TOKEN frame on a previous connection
-to what it believes to be the same server, it can include that value in the
-Token field of its Initial packet.
+If the client has a token received in a NEW_TOKEN frame on a previous
+connection to what it believes to be the same server, it can include that value
+in the Token field of its Initial packet.
```

Comment:
This paragraph looks like a rewrap to a shorter line length than the rest of the document.  The original layout appears to be fine.